agents/synacc/synacc.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+import os
+import requests
+import argparse
+
+on_rtd = os.environ.get('READTHEDOCS') == 'True'
+if not on_rtd:
+    from ocs import ocs_agent, site_config
+    from ocs.ocs_twisted import TimeoutLock
+
+
+class SynAccAgent:
+    def __init__(self, agent, ip_address, username, password):
+        self.agent = agent
+        self.lock = TimeoutLock()
+        self.ip_address = ip_address
+        self.user = username
+        self.passw = password
+
+    def get_status(self, session, params=None):
```

Comment:
It would be helpful on the Client end to have the outlet status saved in `session.data`. This now has documentation here: https://ocs.readthedocs.io/en/develop/developer/session-data.html

Reply:
Will change in the next iteration: see comment below